frontend/catalyst/jax_primitives.py

```diff
@@ -313,7 +313,9 @@ class GradParams:
     method: str
     scalar_out: bool
     h: float
-    argnum: List[int]
+    argnum: Union[int, List]
```

Comment:
@dime10 Not quite sure about the names here: `argnum` is the original argnum, `argnum_list` is the one transformed as a list and `argnum_final` is the the one expanded if args are pytrees. Let me know what you think!

Reply:
I used expanded_argnum and scalar_argnum thanks for the suggestions!